index.php

```diff
@@ -38,12 +38,40 @@
 		</div>
 		<?php endif; ?>
 
+
+
+
  		<div class="pull-right visible-desktop home-lab">
 			<a data-toggle="modal" href="#tutorial_video_modal"><img src="img/screencast_thumb.png"></a>
 		</div>
-		<h1><?php echo $homepage_title; ?> <small><?php echo $homepage_subtitle; ?></small></h1>
 
-		<p class="lead">You can use labrador to find and download processed data or request new datasets.
+<h1><img class="pull-left visible-desktop" src="img/CTR_600.png" style="margin: 0 20px 10px 0;">
+
+		<?php echo $homepage_title; ?> <small><?php echo $homepage_subtitle; ?></small></h1>
+
+		<p class="lead">
+
+        System Status: &nbsp </FONT>
```

Comment:
`&nbsp` should be `&nbsp;`

Where did the `</FONT>` come from? Font elements are _evil_ 👹